config/schedule.rb

```diff
@@ -23,7 +23,11 @@
 set :output, "#{Dir.pwd}/public/cron_log.log"
 
 # To simply print date into the log file for checking if cron job is working properly
-every 1.minutes do
-	puts Dir.pwd
-	command "date -u" #This will print utc time every 1 min in log/cron_log.log file
-end
+# every 1.minutes do
```

Comment:
I think we should not comment this? The only thing it is doing here is printing the time every one minute so that we can make sure that it is running. What do you say?

Reply:
Oh I was testing something, forgot to uncomment it. Will do!